group_vars/customer/eugene/development/main.yml

```diff
@@ -30,3 +30,6 @@ edxapp_theme_tag: "marvel-theme-ginkgo"
 # compile custom configMaps
 forum_host: "forum.{{ project_name}}.{{ domain_name }}"
 forum_port: 9292
+
+# increase edxapp media volume size
+edxapp_media_volume_size: 4Gi
```

Comment:
I think it's the data volume that requires an increase.

Reply:
:+1: